SwiftUIDZ/SwiftUIDZ/PaymentViewModel.swift

```diff
@@ -0,0 +1,86 @@
+// PaymentViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Вью модель экрана оплаты
+final class PaymentViewModel: ObservableObject {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let maxCountChar = 15
+        static let minCountChar = 6
+    }
+
+    // MARK: - Public Properties
+
+    @Published var cardHolderName = ""
+    @Published var cardNumber = ""
+    @Published var cvcNumber = ""
+    @Published var isCvcAlertShown = false
+    @Published var isMonthPickerShown = false
+    @Published var isYearPickerShown = false
+    @Published var selectCardNumberText = "0000 0000 0000 0000"
+    @Published var selectCardHolderName = "Your Name"
+    @Published var text = ""
+    @Published var selectCvcText = "000"
+    @Published var isAddNowAlertShown = false
+    @Published var selectMonthText = "05"
+    @Published var selectYearText = "23"
+    @Published var backDegree = 0.0
+    @Published var frontDegree = -90.0
+    @Published var isFlipped = false
+    @Published var months = ["01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12"]
+    @Published var years = ["23", "24", "25", "26", "27", "28", "29", "30"]
+
+    // MARK: - Private Properties
+
+    private let durationAndDelay: CGFloat = 0.3
+
+    // MARK: - Public methods
+
+    func cvcAlertShow() {
+        if cvcNumber.count < 3 {
```

Comment:
```suggestion
        if cvcNumber.count < 3 {
```
магическое скорее всего